README.md

```diff
@@ -17,15 +17,15 @@ Qri has the capacity to run as a remote built-in, but needs to be configured to
 Before we get into setting up a remote, it helps to see what one can actually do. Here's a quick rundown on pushing to a remote we've already set up:
 
 ```sh
-# install qri if you haven't already:
-curl -fsSL https://qri.io/install.sh
+# install qri if you haven't already. You may need to run with sudo:
+curl -fsSL https://qri.io/install.sh | bash -
```

Comment:
let's move this to a separate step titled "install qri locally"

** **
### Install qri Locally
If you don't already have qri on your machine, you'll need to install
```sh
# install qri if you haven't already. You may need to run with sudo:
curl -fsSL https://qri.io/install.sh		 curl -fsSL https://qri.io/install.sh | bash -

# run qri setup to pick your username
qri setup
```